### ISSUE TITLE
Nominate Nic from Upbound.io as the new OAM spec maintainer

### ISSUE THREAD

**Issue by resouer** [oam-dev/spec#385]
I'd want to nominate @negz from Upbound.io as the OAM spec maintainer based on his significant contribution to the OAM spec release v0.2.0 (previously named v1alpha2).

Note that the OAM spec's new governance model (based on [Community Spec](https://github.com/CommunitySpecification/1.0)) is still WIP (/cc @vturecek ) so we may have to follow the "old school" no-objection based process in this issue page to add Nic into the OWNERS file (unless Nic reject this nomination).

Existing maintainers, @oam-dev/oam-maintainers , please feel free to +1 in the comment if you endorse this nomination, and feel free to raise objection if there's any concern (the nomination would hold in this case).

**Comment by hongchaodeng** [comment]
+1
@negz has done a great job in pushing forward the edge of cloud native application and infrastructure management. He is very active in the community and contributing ideas in the design of OAM and Crossplane. I would give my thumbs up to him 👍

**Comment by vturecek** [comment]
Agreed. Assuming Nic accepts, I'll open a PR to update the owners file.

**Comment by wonderflow** [comment]
+1

**Comment by artursouza** [comment]
+1
